lib/libutils/isoc/bget_malloc.c

```diff
@@ -457,7 +457,17 @@ void raw_free(void *ptr, struct malloc_ctx *ctx, bool wipe)
 	raw_malloc_validate_pools(ctx);
 
 	if (ptr)
-		brel(maybe_untag_buf(ptr), &ctx->poolset, wipe);
+		brel(maybe_untag_buf(ptr), &ctx->poolset, wipe,
+		     false /*!wipe-only*/);
+}
+
+void raw_wipe(void *ptr, struct malloc_ctx *ctx)
+{
+	raw_malloc_validate_pools(ctx);
+
+	if (ptr)
+		brel(maybe_untag_buf(ptr), &ctx->poolset, true /*wipe*/,
```

Comment:
Since the wiping always is going to be done in a separate step there's no need to let the `brel()` function do the wiping.
We do something close to wiping in `maybe_untag_buf()` already. How about extending that function to wipe the buffer too?

Reply:
We don't knon the size of the space, this is where brel gives a size big hint. We wipe it, indeed using internal states. Needs another though. We must find back from bet the initially required size we can safely wipe without writing to brel internal states.

Reply:
What's wrong with `bget_buf_size()`?

Reply:
it was hidden behind some tree. Here i can see it now :)